index.html

```diff
@@ -40,7 +40,147 @@ <h2 class="social order">LET'S CONNECT</h2>
+                </div>
+            </div>
+
+        </section>
+        <section class="section2">
+            <div class="right-block">
+                <img class="image-section2" src=".images/Snapshoot Portfolio1.png" alt="project image">
+            </div>
+            <div class="left-block">
+                <div class="experience1">
+                    <h1 class="section-title1">Multi-Post
+                        Stories</h1>
+                    <ul class="list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list3">2015</li>
+                    </ul>
+                </div>
+                <p class="section-description0">A daily selection of privately personalized reads; no
+                    accounts or sign-ups
+                    required.</p>
+
+                <p class="section-description1">A smart assistant to make driving more safe, efficient, and fun by
+                    unlocking
+                    your most expensive computer: your car.</p>
+                <div class="tags">
+                    <div class="li11">
+                        <article class="text1">html</article>
+                    </div>
+                    <div class="li22">
+                        <article class="text2">css</article>
+                    </div>
+                    <div class="li33">
+                        <article class="text3">javascript
+                    </div>
+                </div>
+                <div class="action">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+
+        </section>
+        <section class="section3">
+            <div class="right-block">
+                <img class="image-section3" src=".images/Snapshoot Portfolio2.png" alt="project image">
+            </div>
+            <div class="left-block">
+                <div class="experience">
+                    <h1 class="section-title">Tonic</h1>
+                    <h1 class="section-titlef">Tonic</h1>
+                    <ul class="list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list3">2015</li>
+                    </ul>
+                </div>
+                <p class="section-description0">A daily selection of privately personalized reads; no
+                    accounts or sign-ups
+                    required.</p>
+                <div class="tags">
+                    <div class="li11">
+                        <article class="text1">html</article>
+                    </div>
+                    <div class="li22">
+                        <article class="text2">css</article>
+                    </div>
+                    <div class="li33">
+                        <article class="text3">javascript
+                    </div>
+                </div>
```

Comment:
- You did a great job by making the list of technology in the ` div ` however I can suggest for the best practices to use  `<ul> and  <li>`  tags  when you structure your list so The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li :+1: